purchase_sale_container_deposit_inter_company/tests/test_inter_company_purchase_sale_container_deposit.py

```diff
@@ -26,6 +26,40 @@ def setUpClass(cls):
         cls.env["product.pricelist"].sudo().search([]).write(
             {"currency_id": cls.env.ref("base.USD").id}
         )
+        # Add inventory and purchase group to user_company_a
+        cls.user_company_a = cls.env["res.users"].write(
+            {
+                "groups_id": [
+                    (
+                        4,
+                        0,
```

Comment:
Could you use the New Command syntax (x4) ? It is lighter.

Reply:
Done, thanks @legalsylvain for looking into it !